include/pingcap/kv/LockResolver.h

```diff
@@ -51,16 +55,19 @@ struct Lock
           ttl(l.lock_ttl()),
           txn_size(l.txn_size()),
           lock_type(l.lock_type()),
-          lock_for_update_ts(l.lock_for_update_ts())
+          use_async_commit(l.use_async_commit()),
+          lock_for_update_ts(l.lock_for_update_ts()),
+          min_commit_ts(l.min_commit_ts())
     {}
 };
 
 using LockPtr = std::shared_ptr<Lock>;
 
 inline std::string getLockInfo(LockPtr & lock)
 {
-    return "key: " + lock->key + " primary: " + lock->primary + " txn_start_ts: " + std::to_string(lock->txn_id) + " lock_for_update_ts: "
-        + std::to_string(lock->lock_for_update_ts) + " ttl: " + std::to_string(lock->ttl) + " type: " + std::to_string(lock->lock_type);
+    return "key: " + lock->key + " primary: " + lock->primary + " txn_start_ts: " + std::to_string(lock->txn_id)
+        + " lock_for_update_ts: " + std::to_string(lock->lock_for_update_ts) + " min_commit_ts: " + std::to_string(lock->min_commit_ts)
+        + " ttl: " + std::to_string(lock->ttl) + " type: " + std::to_string(lock->lock_type);
```

Comment:
Please resolve the conflict, use `Lock:: toDebugString` instead.

Reply:
Done

Reply:
You have made some changes to the function in the previous commits.
I think you should apply those changes to `Lock::toDebugString` instead of just deleting the old function.